.github/workflows/build.yml

```diff
@@ -163,15 +163,15 @@ jobs:
           ! grep "stdout:" stdout.log
 
       - name: build library_search cache
-        run: lake build -KCI MathlibExtras
+        run: lake build -KCI -R MathlibExtras
```

Comment:
This should be in the master yaml file

Reply:
this is not going to be committed, it is just kicking CI (and it didn't work anyway because of https://github.com/leanprover/lake/issues/181). The alternative is the commit I just pushed, which invalidates the cache by adding whitespace in the lake manifest. I do plan to commit that, because it is important that anything downstream of this commit not get old caches because they will poison the new caches in CI.